index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="header-container">
+    <div class="header-item header-item1"><time id="time">9:41</time></div>
+    <div class="header-item header-item2"><img id="signal" src="src/signal.png" alt="signal"></div>
+    <div class="header-item header-item3"><p id="name">Anand Pathak</p></div>
+    <div class="header-item header-item4"><img id="tab" src="src/tab.png" alt="tab"></div>
+  </header>
+
+  <section class="intro-section-container">
+    <p class="intro-section-item" id="intro-text1">Hey There. I'm Anand</p>
+    <p class="intro-section-item" id="intro-text2">I’am a Software Developer</p>
+    <p class="intro-section-item" id="intro-text3">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+      don’t hestiate to contact me.</p>  
+    <ul class="intro-section-item icon-container">
+      <li><button class="icon-element" type="button"><img src="src/github.png" alt="github"></button></li>
+      <li><button class="icon-element" type="button"><img src="src/i.png" alt="i"><img src="src/n.png" alt="n"></button></li>
+      <li><button class="icon-element" type="button"><img src="src/v.png" alt="v"></button></li>
+      <li><button class="icon-element" type="button"><img src="src/t.png" alt="twitter"></button></li>
+      <li><button class="icon-element" type="button"><img src="src/m.png" alt="m"></button></li>
+    </ul>
```

Comment:
This is a good job however i suggest you change the `<button>`s to `<a>' tags. This icons redirect a user to your social pages.
Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href